apis/ml-core/ml-coreapi.yaml

```diff
@@ -0,0 +1,8163 @@
+openapi: 3.0.1
+info:
+  title: Kendra Service API(s)
```

Comment:
@aks30 @mohitga is this a DIKSHA specific API set? Should the title be 'Kendra Service'? Also, I notice that this set has a 'Create User' end point as well. How is it different from the existing Sunbird User APIs? Or is this set replacing the existing user APIs? Please clarify.

Reply:
Hi @Sunitashelar, we will be raising a new PR against the 4.0 release branch. Kindly ignore this for now.